<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRecipeChoiceDlg</class>
 <widget class="QDialog" name="QRecipeChoiceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配方选择器</string>
  </property>
  <widget class="QPushButton" name="m_helpBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="m_tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基本</string>
    </attribute>
    <widget class="QGroupBox" name="m_UseMacroCkkgroupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>脚本</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>26</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>脚本：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_MacroCmBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>22</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_new">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>写脚本</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>显示风格</string>
     </property>
     <widget class="QRadioButton" name="m_ComboradioBtn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>下拉框</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="m_TableradioBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>列表(长按两秒选中)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>551</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>属性</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>241</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>配方组名称：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_Transparentlabel">
         <property name="text">
          <string>透明度：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="m_TransparentspBox">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="m_RowCountlabel">
         <property name="text">
          <string>行数：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="m_RowCountSpBox">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>241</width>
        <height>106</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>字体：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_fontCbBox">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>大小：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="m_FontSizecbBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>文本色：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="m_TextColorpushBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>背景色：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="m_BackColorpushBtn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
